lambda/voting/contracts/imports/MetaKeepLambda.sol

```diff
@@ -0,0 +1,732 @@
+// SPDX-License-Identifier: MIT 
+pragma solidity ^0.8.0;
```

Comment:
let's also create a top level folder lambda/imports
that we will use as the master import folder

Reply:
done